career-readiness.md

```diff
@@ -6,16 +6,16 @@ The LEARN program focuses on readying our students for a career in technology. T
 
 Deploying a project is an exciting, yet often challenging process. There are usually small errors that cause deployments to fail or cause bugs in production. One solution to this is to deploy early and update the deployment as the code evolves.
 
-Submit the link to your deployed portfolio site:
+Submit the link to your deployed portfolio site: https://marshall2309.github.io/
 
 We recommend GitHub pages for HTML/CSS and Render.com for React. Your portfolio site should display AT LEAST "Hello World" text. Completely blank sites will be considered incomplete. 
 
 ### LinkedIn
 
-Your LinkedIn url:
+Your LinkedIn url: https://www.linkedin.com/in/chris-k-marshall/
 
 Check the boxes below to certify that you have completed the following items on LinkedIn:
 
-- [ ] create a new post about your first five weeks at LEARN. Topics can include (but are not limited to) Ruby, sharing a project you created, sharing a resource you found helpful, or portfolios.
-- [ ] add at least ten more professional connections. You should have at least 50 LinkedIn connections.
-- [ ] react (like, celebrate, comment, etc) to ten posts.
+- [X] create a new post about your first five weeks at LEARN. Topics can include (but are not limited to) Ruby, sharing a project you created, sharing a resource you found helpful, or portfolios.
+- [X] add at least ten more professional connections. You should have at least 50 LinkedIn connections.
```

Comment:
Gotta say, I am really impressed with the amount of LinkedIn connections you have!  👏